src/main/menu/electron-menu-items.injectable.ts

```diff
@@ -29,7 +29,7 @@ const electronMenuItemsInjectable = getInjectable({
     const extensions = di.inject(mainExtensionsInjectable);
 
     return computed(() =>
-      extensions.get().flatMap((extension) => extension.appMenus),
+      extensions.get().flatMap((extension) => extension.appMenus.slice()),
```

Comment:
What is the reasoning behind `.slice()`?

Reply:
It was for supporting observable arrays but it's not actually needed. good catch!